src/cluster_legacy.c

```diff
@@ -370,7 +419,10 @@ int auxTlsPortPresent(clusterNode *n) {
 typedef struct {
     size_t totlen; /* Total length of this block including the message */
     int refcount;  /* Number of cluster link send msg queues containing the message */
-    clusterMsg msg;
+    union {
+        clusterMsg msg;
+        clusterMsgLight light_msg;
```

Comment:
nit - match the struct name?
```suggestion
        clusterMsgLight msg_light;
```